smithy-model/src/main/java/software/amazon/smithy/model/loader/ModelValidator.java

```diff
@@ -103,34 +108,43 @@ static ValidatorFactory defaultValidationFactory() {
 
     static final class Builder implements SmithyBuilder<ModelValidator> {
 
-        private final List<Validator> validators = new ArrayList<>();
-        private final List<ValidationEvent> includeEvents = new ArrayList<>();
+        private final BuilderRef<List<Validator>> validators = BuilderRef.forList();
```

Comment:
I made minor tweaks to update this builder to use the builder ref API so that we don't have to make unnecessary copies.